### PR TITLE
Set Private to false for specific projects instead of everything.

### PR DESCRIPTION
**Pull request by jlaanstra**
Set Private explicitly to false for certain projects instead of setting it to false for everything and then setting it true for certain projects and allow projects to opt-out.

TODO:

- [x] Add docs for the new property.

### CODE REVIEW

**Comment by kennykerr** [comment]
@jlaanstra "/azp run" is no longer required.

**Comment by kennykerr** [comment]
Maybe add documentation for what this is for here: https://github.com/microsoft/cppwinrt/tree/master/nuget

**Comment by kennykerr** [comment]
Is this PR still active?

**Comment by jlaanstra** [comment]
@kennykerr yes it is. I still need to add docs for the property.

File: nuget/CppWinrtRules.Project.xml [hunk 1]
```diff
@@ -76,4 +76,9 @@
                 Description="Enables or disables the generation of Windows Metadata"
                 Category="General" />
 
+  <BoolProperty Name="CppWinRTEnableDefaultPrivateFalse"
+                DisplayName="Enable Copy Local Defaults"
+                Description="Enables or disables the default for copying bineries to the output folder"
```
Comment (jlaanstra) [inline nuget/CppWinrtRules.Project.xml:81]:
```suggestion
                Description="Enables or disables the default for copying binaries to the output folder"
```

**Comment by jlaanstra** [comment]
@kennykerr this is ready to merged.

**Comment by kennykerr** [comment]
Thanks!

**Pull request merged**